task8-mod2/src/main/java/com/example/classloader/implement/CalculateMultiply.java

```diff
@@ -0,0 +1,39 @@
+package com.example.classloader.implement;
+
+import com.example.classloader.Calculable;
+
+import java.util.ArrayList;
+
+public class CalculateMultiply implements Calculable {
+    private final String description = "The multiplication of all parameters";
+
+    private ArrayList<Double> parametersList;
+
+    public CalculateMultiply(ArrayList<Double> parametersList) {
+        this.parametersList = parametersList;
+    }
+
+    @Override
+    public int getParametersCount() {
+        return this.parametersList.size();
+    }
+
+    @Override
+    public String getDescription() {
+        return this.description;
+    }
+
+    @Override
+    public double getResult() {
+        double result = 0;
+        if (!parametersList.isEmpty()) {
+            result = parametersList.get(0);
+            if (parametersList.size() > 1) {
```

Comment:
hm ... Multiplication can work with 2 values only.. getParametersCount should be set explicit to 2 here...

Actually, there unary operations, binary operation, ternary operations (not mathematical)

and we need to ensure that passed parametersList has size of applicable xNary operation parameters count. Otherwise exception need to be thrown